README.md

```diff
@@ -1,223 +1,113 @@
+which is still in alpha, and does not intend to add QUIC functionality to 1.1.x.
+There is a community need for a QUIC capable TLS library. This fork is intended
+as stopgap solution to enable higher level frameworks and runtimes to use QUIC
+with the proven and reliable TLS functionality from OpenSSL. This fork will be
+maintained until OpenSSL officially provides reasonable support for QUIC
+implementations.
 
-However, for a large variety of operating systems precompiled versions
-of the OpenSSL toolkit are available. In particular on Linux and other
-Unix operating systems it is normally recommended to link against the
-precompiled shared libraries provided by the distributor or vendor.
+This fork can be considered a supported version of
+[OpenSSL PR 8797](https://github.com/openssl/openssl/pull/8797).
+We will endeavor to track OpenSSL releases within a day or so, and there is an
+item below about how we'll follow their tagging.
 
-For Testing and Development
----------------------------
+On to the questions and answers.
 
-Although testing and development could in theory also be done using
-the source tarballs, having a local copy of the git repository with
-the entire project history gives you much more insight into the
-code base.
+What about branches?
+--------------------
 
-The official OpenSSL Git Repository is located at [git.openssl.org].
-There is a GitHub mirror of the repository at [github.com/openssl/openssl],
-which is updated automatically from the former on every commit.
+We don't want to conflict with OpenSSL branch names. Our current plan is to append
+`+quic`. Release tags are likely to be the QUIC branch with `-releaseX` appended.
+For example, the OpenSSL tag `openssl-3.0.0-alpha12` would have a branch named
+`openssl-3.0.0-alpha12+quic` and a release tag of `openssl-3.0.0-alpha12+quic-release1`
 
-A local copy of the Git Repository can be obtained by cloning it from
-the original OpenSSL repository using
+How are you keeping current with OpenSSL?
+-----------------------------------------
 
-    git clone git://git.openssl.org/openssl.git
+(In other words, "What about rebasing?")
 
-or from the GitHub mirror using
+Our plan is to always rebase on top of an upstream release tag. In particular:
 
-    git clone https://github.com/openssl/openssl.git
+- The changes for QUIC will always be at the tip of the branch -- you will know what
+  is from the original OpenSSL and what is for QUIC.
+- New versions are quickly created once upstream creates a new tag.
+- The use of git commands (such as "cherry") can be used to ensure that all changes
 
-If you intend to contribute to OpenSSL, either to fix bugs or contribute
-new features, you need to fork the OpenSSL repository openssl/openssl on
-GitHub and clone your public fork instead.
+have moved forward with minimal or no changes. You will be able to see
+QUIC: Add X" on all branches and the commit itself will be nearly identical on all
```

Comment:
(We should probably add the missing double-quote mark at the start of the line in a follow-up)

Reply:
Fixed in tip commit.